Samples/Dinner/Sources/Onboarding/main.swift

```diff
@@ -0,0 +1,63 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OpenTelemetry open source project
+//
+// Copyright (c) 2021 Moritz Lang and the Swift OpenTelemetry project authors
```

Comment:
Authorship I believe you'd be fine with the samples using this repo's copyright @slashmo ?